digdag-docs/src/command_reference.rst

```diff
@@ -512,6 +512,8 @@ Examples:
 
 :command:`--resume`
   Retry only failed tasks. Successfully finished tasks are skipped.
+  Note that if a retry is set up at the group level (not task level), it will resume from the beginning of the group, not only failed tasks.
+  The same applies to the nested group.
```

Comment:
A bit uncertain.
I am not sure, but the following may be better.
> If __retry_ is set to each nested groups, it will resume from the top of the group.